### DISCUSSION TITLE
change bevy text live

### DISCUSSION THREAD

**Discussion by ZATTYAMOND** [bevyengine/bevy#9148] [Q&A]
i wanna change my bevy text live i am using this example i am using 0.11 here is the example https://bevyengine.org/examples/ui/text/ how can i pls show me a code example

**Comment by rlidwka** [comment]
At the bottom of that example, there is a system that literally changes text:

```rs
fn text_update_system(
    diagnostics: Res<DiagnosticsStore>,
    mut query: Query<&mut Text, With<FpsText>>,
) {
    for mut text in &mut query {
        if let Some(fps) = diagnostics.get(FrameTimeDiagnosticsPlugin::FPS) {
            if let Some(value) = fps.smoothed() {
                // Update the value of the second section
                text.sections[1].value = format!("{value:.2}");
            }
        }
    }
}
```

Is that what you are looking for? Or do you want to get user-editable Input?
